src/components/label-with-arrow/label-with-arrow.module.scss

```diff
@@ -8,7 +8,7 @@
     font: var(--heading5);
     font-size: 15px;
     letter-spacing: 0.04em;
-    color: var(--colorA1);
+    color: var(--colorA5);
```

Comment:
The default label and arrow icon color changed to `--colorA5` (almost black). Why? Because  in previous version when user adds `TextBanner` component to the page, they don't see a link at all. It's color was the same as background. Let it rather be black.

Reply:
Why don't they see it? Remove color from here.